README.md

```
@@ -116,7 +116,8 @@ kfk-gift-registry/
       <img src="https://img.shields.io/badge/💻_engineer-27AE60?style=flat-square"/>
     </td>
     <td align="center" width="160">
-      <img src="assets/team/eddie.jpg" width="110" style="border-radius:10%;"/><br/>
+      <a href="https://www.linkedin.com/in/edwardrsong/" target="_blank" rel="noreferrer noopener">
+      <img src="assets/team/eddie-song.jpg" width="110" style="border-radius:10%;"/><br/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)